tck/src/main/java/org/eclipse/microprofile/health/tck/EnforceQualifierTest.java

```diff
@@ -69,18 +61,8 @@ public void testFailureResponsePayload() throws Exception {
         JsonObject json = jsonReader.readObject();
         System.out.println(json);
 
-        // response size
-        JsonArray checks = json.getJsonArray("checks");
-        Assert.assertEquals(checks.size(), 1, "Expected a single check response");
-
-        // single procedure response
-        JsonValue check = checks.get(0);
-
-        Assert.assertEquals(
-                asJsonObject(check).getString("name"),
-                "check-with-annotation",
-                "Expected a CDI health check to be invoked, but it was not present in the response"
-        );
+        Assert.assertEquals(json.getString("outcome"), "UP","Expected outcome UP");
```

Comment:
"outcome: should be changed to "status"

Reply:
I think it is better to assert `check-without-annotation` does not exist in the payload.

Reply:
Test was resurected from #119 reversion, reason why there was this "outcome" remaining.
I fixed that and I propose that test enhancement should be proposed in the RC1